src/includes/getting-started-config/java.servlet.mdx

```diff
@@ -0,0 +1,61 @@
+The following example configures a `SentryInitializer` servlet container initializer that initializes Sentry on application startup.
+
+```java
+package sentry.sample;
+
```

Comment:
```suggestion
```

Reply:
Since this package is referenced from the properties file down below i believe it should stay